index.js

```diff
@@ -40,10 +46,14 @@ exports.run = function(runner, specs) {
         } catch (err) {
           reject(err);
         }
-      });
+      }
     });
   });
 
+  function isCucumber2() {
+    return !!Cucumber.defineSupportCode;
```

Comment:
This could break if they change `defineSupportCode` function, is there a way to put a check based on cucumber versions 1 & 2? just trying to understand